tests/test_floorist.py

```diff
@@ -177,3 +177,13 @@ def test_floorplan_with_one_failing_dump(self, caplog, session):
         assert 'ProgrammingError' in caplog.text
         assert 'Dumped 1 from total of 2'
         assert wr.s3.list_directories(prefix, boto3_session=session) == [f"{prefix}/numbers/"]
+
+    def test_floorplan_valid(self, caplog, session):
+        prefix = f"s3://{env['AWS_BUCKET']}"
+        env['FLOORPLAN_FILE'] = 'tests/floorplan_valid.yaml'
+        main()
+        assert 'Dumped 1 from total of 1'
```

Comment:
Ups, I just spotted this. I think this line should be asserting based on the output of the command.

Reply:
I don't understand what are you saying :disappointed:

Reply:
I'm sorry, what I mean is ... 

```python
assert "some-string"
```

Is essentially doing nothing . I think you wanted to do something like 

```
assert 'Dumped 1 from total of 1' in some_output 
```

Reply:
That's affecting the whole test suite, let's solve it outside of this PR.

Reply:
I'll fix this too with https://github.com/RedHatInsights/floorist/pull/27/files